### DISCUSSION TITLE
ESP32Cam blinking fast all the time

### DISCUSSION THREAD

**Discussion by mido1978** [jomjol/AI-on-the-edge-device#814] [Q&A]
Hi, I did all the necessary installation steps fro the wiki. I use a MacBook Pro, MicroSD Card (Raspberry and SanDisk; both 64Gb), formatted with FAT32. But nevertheless, the board is blinking fast all the time. I even plugged a Anker power supply. 

No idea, what to try next. It is strange…

Any suggestions/ideas what might be wrong?

Thanks!

**Comment by jomjol** [comment]
Please read here: https://github.com/jomjol/AI-on-the-edge-device/wiki/Installation#sd-card

![grafik](https://user-images.githubusercontent.com/30766535/170361532-0ecc128f-7530-4c53-badd-e68a12556eec.png)

RTFM ;-)

**Reply by mido1978** [reply]
@jomjol Yes I did and tried 2 different cards. I ordered another one now and will report here.

**Comment by jomjol** [comment]
Can you post a serial log during the startup of the esp32 until it fails? This can give some more hints, if it's the sd-card.

**Comment by mido1978** [comment]
> Can you post a serial log during the startup of the esp32 until it fails? This can give some more hints, if it's the sd-card.

How is this done? I am going to search for How-To do it and will report here. 
Thanks!

**Comment by jomjol** [comment]
You can do this easy with an arduino ide or with any serial log interface.
